FoodTracker/Models/Meal.swift

```diff
@@ -0,0 +1,57 @@
+import UIKit
+import os.log
+
+class Meal: NSObject, NSCoding {
+    
+    //MARK: -Properties
```

Comment:
I like to write marks like this way:
// MARK: - Properties  (If you want to put a separator)
// MARK: Properties (Else)
Looks cool to me that way.
But the most important this is consistency.